plugins/modules/cloudformation_stack_set.py

```diff
@@ -84,7 +84,7 @@
   capabilities:
     description:
     - Capabilities allow stacks to create and modify IAM resources, which may include adding users or roles.
-    - Currently the only available values are 'CAPABILITY_IAM' and 'CAPABILITY_NAMED_IAM'. Either or both may be provided.
+    - Currently the only available values are 'CAPABILITY_IAM' and 'CAPABILITY_NAMED_IAM'. Either or ALL may be provided.
     - >
         The following resources require that one or both of these parameters is specified: AWS::IAM::AccessKey,
         AWS::IAM::Group, AWS::IAM::InstanceProfile, AWS::IAM::Policy, AWS::IAM::Role, AWS::IAM::User, AWS::IAM::UserToGroupAddition
```

Comment:
Can you also please add a note like:

`Support for C(CAPABILITY_AUTO_EXPAND) was added in release 6.1.0.`